libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.kt

```diff
@@ -437,37 +462,64 @@ open class PushService {
         protected const val UNREGISTRATION_STATUS_SUCCEEDED = 2
         protected const val UNREGISTRATION_STATUS_FAILED = 3
 
+        /**
+         * The Android background tasks name of the push notifications
+         * registration work request
+         */
+        private const val PUSH_NOTIFICATIONS_REGISTRATION_WORK_NAME = "SalesforcePushNotificationsRegistrationWork"
+
         /**
          * Enqueues a change to one or more user accounts' push notifications
          * registration as persistent work via Android background tasks.
          *
-         * @param userAccount       The user account or null for all user accounts
-         * @param action            The push notifications registration action
-         * @param delayMilliseconds The amount of delay before the push registration
-         * action is taken, such as in a retry scenario
+         * @param userAccount The user account or null for all user accounts
+         * @param action The push notifications registration action
+         * @param delayDays For registration actions, the interval in days
+         * between periodic registration or null for immediate one-time
+         * registration
          */
         internal fun enqueuePushNotificationsRegistrationWork(
             userAccount: UserAccount?,
             action: PushNotificationsRegistrationAction,
-            delayMilliseconds: Long?
+            delayDays: Long?
```

Comment:
Days or hours?

Reply:
I went for days here since this value is private to our SDK and always set to six days.